api/HTMLOptionsCollection.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "6.1"
```

Comment:
`HTMLOptionsCollection` is used for `select.options`, and https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/options#Browser_compatibility says that's supported since earlier than this. http://mdn-bcd-collector.appspot.com/tests/api/HTMLOptionsCollection is testing the prototype only, can you double check if this shouldn't be supported earlier?

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/8727, closing this PR.